Packs/Darktrace/Integrations/Darktrace/Darktrace.yml

```
@@ -8,74 +11,80 @@ configuration:
   name: url
   required: true
   type: 0
+  section: Connect
 - display: Fetch incidents
   name: isFetch
   required: false
   type: 8
+  section: Collect
 - display: Trust any certificate (not secure)
   name: insecure
   required: false
   type: 8
+  section: Connect
+  advanced: true
 - display: Use system proxy settings
   name: proxy
   required: false
   type: 8
+  section: Connect
+  advanced: true
 - display: Incident type
   name: incidentType
   required: false
   type: 13
-- additionalinfo: Public token obtained by creating an API token pair on the /config
-    configuration page
+  section: Connect
+- additionalinfo: Public token obtained by creating an API token pair on the /config configuration page
   displaypassword: Public API Token (Recommended)
   hiddenusername: true
   name: public_creds
   required: false
   type: 9
-- additionalinfo: Private token obtained by creating an API token pair on the /config
-    configuration page
+  section: Connect
+- additionalinfo: Private token obtained by creating an API token pair on the /config configuration page
   displaypassword: Private API Token (Recommended)
   hiddenusername: true
   name: private_creds
   required: false
   type: 9
+  section: Connect
 - additionalinfo: Minimum Darktrace score for fetched incidents (0-100)
   defaultvalue: '0'
   display: Minimum Score
   hidden: false
   name: min_score
   required: true
   type: 0
+  section: Collect
 - additionalinfo: Maximum number of Darktrace model breaches to fetch at a time
   defaultvalue: '50'
   display: Maximum Model Breaches per Fetch
   hidden: false
   name: max_fetch
   required: false
   type: 0
-- additionalinfo: Time to start fetching the first incidents. Default is to begin
-    fetching 1 day ago. Max number of model breaches that will be populated upon first
-    fetch is 20.
+  section: Collect
+- additionalinfo: Time to start fetching the first incidents. Default is to begin fetching 1 day ago. Max number of model breaches that will be populated upon first fetch is 20.
   defaultvalue: 1 day ago
   display: First fetch time
   hidden: false
   name: first_fetch
   required: false
   type: 0
+  section: Collect
 - additionalinfo: Use the "Public API Token (Recommended)" parameter instead.
   display: Public API Token (Deprecated)
   name: public_api_token
   required: false
   type: 4
+  section: Connect
```

```suggestion
  section: Connect
  advanced: true
```
I think it should be advanced because it is deprecated